### DISCUSSION TITLE
Invoke-PnPSiteTemplate

### DISCUSSION THREAD

**Discussion by marcorocca1973** [pnp/syntex-samples#9] [Q&A]
I try to add model template in this repository using PnP 1.8.0. I'm tenant and SharePoint Admin (this is my test tenant), content center exist and with same user I can create and deploy models.
But when I try to import for example PurchaseOrder.pnp or TradeConfirmation.pnp nothing happen :(

What I'm doing wrong?

Thanks for your support!
Marco

**Comment by jameseccles** [comment]
Could you share the command that you are running? Do you get any error in the PowerShell console?
